addons/mass_mailing/static/src/iframe/mass_mailing_iframe.js

```diff
@@ -224,6 +225,33 @@ export class MassMailingIframe extends Component {
         this.state.ready = true;
     }
 
+    /**
+     * As no plugins are loaded in readonly mode, we manually set inlining attributes to
+     * any t-element that might be present in the document, provided its children are inline
+     * (which should always be the case for mass_mailing).
+     * TODO: move this to a readonly plugin once they are implemented
+     * @param {HTMLElement} iframeRef
+     */
+    fixInlineDynamicPlaceholders(iframeRef) {
```

Comment:
Let's rename the function parameter `iframe`, as the `iframeRef` is a Ref object, with a `el` property, which gives the element :)